openlibrary/plugins/openlibrary/js/autocomplete.js

```diff
@@ -4,25 +4,20 @@
     // author-autocomplete
     $.fn.author_autocomplete = function(options) {
         var local_options = {
+            addnew: true,
+
             minChars: 2,
             max: 11,
-            addnew: true,
+            matchSubset: false,
             autoFill: false,
             formatItem: function(item) {
-                var date = '';
-                if (item.key != "__new__") {
```

Comment:
Was it significant if item.key != "__new__"?

Reply:
Sort of; when `item.key == "__new__"`, both dates would have to be undefined, so we get the same effect. Removing it looked a little simpler to me since there's less nesting and since it doesn't really matter whether `item.key` is `__new__` when creating the lifespan string, so less mental load for future programmers reading this code :)